modules/shared/src/main/scala/org/tessellation/json/JsonBinarySerializer.scala

```diff
@@ -1,12 +1,13 @@
 package org.tessellation.json
 
+import scala.util.Try
+
 import io.circe.jawn.JawnParser
 import io.circe.syntax._
 import io.circe.{Decoder, Encoder}
-
 object JsonBinarySerializer {
-  def serialize[A: Encoder](content: A): Array[Byte] =
-    content.asJson.noSpaces.getBytes("UTF-8")
+  def serialize[A: Encoder](content: A): Either[Throwable, Array[Byte]] =
+    Try(content.asJson.noSpaces.getBytes("UTF-8")).toEither
```

Comment:
What exception are we catching here?

Reply:
I guess it could be a problem if we were specifying charset dynamically. If it's hardcoded right here it shouldn't ever throw I think (given that it worked once, why wouldn't it work every other time). Still maybe it's safer to wrap it anyway.

Reply:
Exactly, we have hardcoded encoding so `UnsupportedEncodingException` isn't going to be thrown. The default semantics for any function in our codebase is that the argument is not null, so `NullPointerException` isn't an option as well (otherwise everywhere we would have to check for nulls). Therefore this change is not needed.